src/main/java/io/reactivex/rxjava3/core/Flowable.java

```diff
@@ -12710,6 +12710,45 @@ public final Completable ignoreElements() {
         return RxJavaPlugins.onAssembly(new FlowableOnBackpressureReduce<>(this, reducer));
     }
 
+    /**
+     * The operator applies {@code reducer} to each emitted item from upstream. This operator uses {@code supplier} to
+     * create new items of type R to pass it with value of type T from upstream to {@code reducer} or in case if the downstream is not ready to receive
+     * new items (indicated by a lack of {@link Subscription#request(long)} calls from it) it uses the latest returned value
+     * from {@code reducer} for previous not consumed item from upstream.
+     * When the downstream becomes ready it emits the latest value returned by {@code reducer} to the downstream.
```

Comment:
This is not a good first sentence to describe what the operator is, and I'd probably not write down these details at all. Compare to the previous `onBackpressureReduce`:

> Reduces a sequence of two not emitted values via a function into a single value if the downstream is not ready to receive
> new items (indicated by a lack of {@link Subscription#request(long)} calls from it) and emits this latest
> item when the downstream becomes ready.

Reply:
> Reduces upstream values into an aggregate value, provided by a supplier and combined via a reducer function, while the downstream is not ready to receive items, then emits this aggregate value when the downstream becomes ready.
>
> ...image...
>
> Note that even if the downstream is ready to receive an item, the upstream item will always be aggregated into the output type, calling both the supplier and the reducer to produce the output value.

Reply:
Thanks, changed